docs/architecture/clients/index.md

```diff
@@ -15,13 +15,55 @@ The mono-repository root directory contains three main folders.
 - `bitwarden_license` - Bitwarden Licensed version of the web vault.
 - `libs` - Shared code between the different applications.
 
-`libs` contains the following projects.
+## Libs
 
-- `Common` - Common code shared between all the clients including CLI.
-- `Angular` - Angular specific code used by all the visual clients.
-- `Components` - Angular Components Library.
-- `Node` - Used to be shared code for CLI and Directory Connector CLI, but since directory connector
+Historically, much of the feature logic of Bitwarden has existed within the `apps` directories, only
+being moved to `libs` (usually `libs/common` or `libs/angular`) if it needed to be shared between
+multiple clients.
+
+We are attempting to move to a more modular architecture by creating additional libs that are more
+feature-focused and more accurately convey team code ownership.
+
+- `common` - Common code shared between all the clients including CLI.
```

Comment:
⛏️ I like double-dashing or using colons for this kind of list item separation. We also try to end bulleted lists without periods.